.github/workflows/kb_sdk-actions.yml

```diff
@@ -111,19 +109,9 @@ jobs:
           repository: kbaseapps/${{ matrix.app }}
           path: kbase_app
 
-      - name: prep for repo tests
```

Comment:
Shouldn't a test where the test.cfg file is exercised be included?

Reply:
Done.